lib/ansible/modules/cloud/vmware/vmware_export_ovf.py

```diff
@@ -78,9 +78,9 @@
     type: bool
   download_timeout:
     description:
-    - The user defined timeout in minute of exporting file.
-    - If the vmdk file is too large to export in 10 minutes, specify the value larger than 10, the maximum value is 60.
-    default: 10
+    - The user defined timeout in second of exporting file.
+    - If the vmdk file is too large, you can increase the value.
```

Comment:
I think we should add a note so that the user will get to know that timeout is changed from 10 to 30.